tests/foreman/cli/test_settings.py

```diff
@@ -457,3 +458,42 @@ def test_positive_failed_login_attempts_limit(setting_update, target_sat):
     assert target_sat.execute(f'hammer -u {username} -p {password} user list').status == 0
     target_sat.cli.Settings.set({'name': 'failed_login_attempts_limit', 'value': '0'})
     assert target_sat.cli.Settings.info({'name': 'failed_login_attempts_limit'})['value'] == '0'
+
+
+@pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
+def test_positive_get_setting_info(setting_update, module_target_sat):
+    """Get setting info for a existing setting
+
+    :id: 73568bf2-8419-11f0-9102-c6fadc44396c
+
+    :parametrized: yes
+
+    :expectedresults: Setting info is returned successfully
+
+    :CaseAutomation: Automated
+    """
+
+    setting_info = module_target_sat.cli.Settings.info({'id': 'login_text'}, output_format='json')
```

Comment:
For info method we have default output format as json, so `output_format='json'` can be ommited